test/verify.test.js

```diff
@@ -210,7 +210,7 @@ test('invalid date', function(t) {
 // test values from spec for simple test
 test('valid rsa from spec default', function(t) {
   server.tester = function(req, res) {
-    console.log('> [SIMPLE]', req.headers.authorization);
```

Comment:
I think that logging was mine?  IIRC, it was left in to make it easy to compare test vectors between the spec text and test suite.  There is probably a better way to do it via some test command flag or env var but simple logging worked fine at the time.